crates/subspace-runtime/src/lib.rs

```diff
@@ -1225,3 +1225,14 @@ impl_runtime_apis! {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{Runtime, SubspaceBlockWeights as BlockWeights};
+    use subspace_runtime_primitives::tests_utils::FeeMultiplierUtils;
+
+    #[test]
+    fn multiplier_can_grow_from_zero() {
+        FeeMultiplierUtils::<Runtime, BlockWeights>::multiplier_can_grow_from_zero()
+    }
+}
```

Comment:
Why not putting this test into `subspace-runtime-primitives`?

Reply:
Don't we need the Runtime and BlockWeights from the actual runtime. I'm not too excited about pulling the entire runtime crates into the primitives crates and it was causing cyclic dependencies with other crates. 

Maybe you know of a better way to do that 🤔

Reply:
I'm not ure we need Subspace runtime to run this particular test though. I think it should be possible to construct runtime with just frame system, kind of like we do in mocks. Or am I missing something important?

Reply:
No, that defeats the purpose of testing this if we mock this and runtime endup using something else. I was aiming to actually test this using the given runtime's FeeMultipler